### ISSUE TITLE
ONS record still shown after transferring

### ISSUE THREAD

**Issue by 144** [oxen-io/oxen-electron-gui-wallet#287]
**Describe the bug**

ONS is still shown in "OXEN NAME SERVICE" tab even tho it was transferred. At first it wasn't showing, but when i checked after a few days its there.

**To Reproduce**

Create Session ID ONS -> Transfer it to another Session ID and "OWNER" -> Check "OXEN NAME SERVICE" tab after 10+ confirmations.

**Other information:**

* Device: PC
* OS: Windows 10
* Version: 1.6.0

I'm using "Remote Daemon Only".

**Comment by kylezs** [comment]
@darcys22 Could this be a backend issue? My best guess is that the local cache isn't cleared on a transfer and so it still shows on the list.

**Comment by darcys22** [comment]
Yeah thats my guess too, think its on the oxen-electron-gui-wallet side.
